sections/node.zsh

```diff
@@ -20,12 +20,18 @@ SPACESHIP_NODE_COLOR="${SPACESHIP_NODE_COLOR="green"}"
 # ------------------------------------------------------------------------------
 
 # Show current version of node, exception system.
-spaceship_node() {
+spaceship_async_job_load_node() {
   [[ $SPACESHIP_NODE_SHOW == false ]] && return
-
   # Show NODE status only for JS-specific folders
   [[ -f package.json || -d node_modules || -n *.js(#qN^/) ]] || return
 
+  async_job spaceship spaceship_async_job_node "$PWD" "$PATH"
+}
+
+spaceship_async_job_node() {
+  [[ $SPACESHIP_NODE_SHOW == false ]] && return
```

Comment:
this line is probably not needed, as async job would not be even scheduled if SHOW is set to false, correct?

Reply:
You are correct. That was probably me forgetting to delete it when moving stuff around.

Reply:
Removed